src/services/firebase.js

```diff
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/* global path */
```

Comment:
What does this do?

Reply:
Avoids the linting error, since path isn't defined.

Reply:
Which linter are you using?

Reply:
eslint

Reply:
My fault. We have Firebase configuration in two locations. Would `import 'path'` fix this? In `get-data.js` it's just using `const path = require('path');` but that should be updated to an import as well, I think.

Reply:
Yeah — that should work (and makes sense).